bitbot/nick.go

```diff
@@ -0,0 +1,24 @@
+package bitbot
+
+import (
+	"strings"
+
+	"github.com/whyrusleeping/hellabot"
+)
+
+var NickTrigger = NamedTrigger{
+	ID: "nick",
+	Condition: func(irc *hbot.Bot, m *hbot.Message) bool {
+		var comesFromHost = (m.From == strings.Split(irc.Host, ":")[0])
```

Comment:
Comment this var to explain what you're doing. I was able to read it, but it wasn't immediately obvious you were expecting a bool